plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/helper/SourceContainerAwareDependencyTraverser.xtend

```diff
@@ -17,31 +17,30 @@ import com.google.common.collect.ImmutableList
 import java.util.Collection
 import org.eclipse.n4js.n4mf.ProjectType
 import org.eclipse.n4js.projectModel.IN4JSProject
-import org.eclipse.n4js.projectModel.IN4JSSourceContainerAware
 import org.eclipse.n4js.utils.DependencyTraverser
 
 /**
- * Class for traversing {@link IN4JSSourceContainerAware source container aware} dependencies and
+ * Class for traversing {@link IN4JSProject source container aware} dependencies and
  * indicating cycles in the dependency graph.
  */
-class SourceContainerAwareDependencyTraverser extends DependencyTraverser<IN4JSSourceContainerAware> {
+class SourceContainerAwareDependencyTraverser extends DependencyTraverser<IN4JSProject> {
```

Comment:
Can it be named `ProjectDependencyTraverser`?
Do we still need generic `DependencyTraverser`?